providers/azure/README.md

```diff
@@ -273,6 +261,155 @@ To verify that virtual kubelet has started, run:
+ 
+```console
+{
+  "appId": "bef76eb3-d743-4a97-9534-03e9388811fc",
+  "displayName": "azure-cli-2018-08-29-22-29-29",
+  "name": "http://azure-cli-2018-08-29-22-29-29",
+  "password": "1d257915-8714-4ce7-xxxxxxxxxxxxx",
+  "tenant": "72f988bf-86f1-41af-91ab-2d7cd011db48"
+}
+```
+Save the output values from the command output in enviroment variables. 
+
+```cli
+export AZURE_TENANT_ID=<Tenant>
+export AZURE_CLIENT_ID=<AppId>
+export AZURE_CLIENT_SECRET=<Password>
+```
+
+These values can be integrated into the `az aks create` as a field ` --service-principal $AZURE_CLIENT_ID \`.
+
+### Integrating Azure VNet Resource
+
+If you want to integrate an already created Azure VNet resource with your AKS cluster than follow these steps. 
+Grab the virtual network resource id with the following command:
+
+```cli
+az network vnet show --resource-group $AKS_CLUSTER_RG --name $VNET_NAME --query id -o tsv
+```
+
+Grant access to the AKS cluster to use the virtual network by creating a role and assigning it. 
+
+```cli
+az role assignment create --assignee $AZURE_CLIENT_ID --scope <vnetId> --role NetworkContributor
+```
+
+### Create an AKS cluster with a virtual network
+
+Grab the id of the cluster subnet you created earlier with the following command. 
+
+```cli
+az network vnet subnet show --resource-group $AKS_CLUSTER_RG --vnet-name $VNET_NAME --name $CLUSTER_SUBNET_NAME --query id -o tsv
+```
+
+Save the entire output starting witn "/subscriptions/..." in the following enviorment variable. 
+
+```cli 
+export VNET_SUBNET_ID=<subnet-resource>
+```
+
+Use the following command to create an AKS cluster with the virtual network you've already created. 
+
+```cli
+az aks create \
+    --resource-group myResourceGroup \
+    --name myAKSCluster \
+    --node-count 1 \
+    --network-plugin azure \
+    --service-cidr 10.0.0.0/16 \
+    --dns-service-ip $KUBE_DNS_IP \
+    --docker-bridge-address 172.17.0.1/16 \
+    --vnet-subnet-id $VNET_SUBNET_ID \
+    --client-secret $AZURE_CLIENT_SECRET
+```
+
+### Deploy Virtual Kubelet
+
+Manually deploy the Virtual Kubelet, the following env. variables have already been set earlier. You do need to pass through the subnet you created for ACI earlier, otherwise the container instances will not be able to participate with the other pods within the cluster subnet. 
+
+```cli
+RELEASE_NAME=virtual-kubelet
+NODE_NAME=virtual-kubelet
+CHART_URL=https://github.com/virtual-kubelet/virtual-kubelet/raw/master/charts/$VK_RELEASE.tgz
+```
+
+If your cluster is an AKS cluster: 
+```cli
+helm install "$CHART_URL" --name "$RELEASE_NAME" \
+  --set provider=azure \
+  --set providers.azure.targetAKS=true \
+  --set providers.azure.vnet.enabled=true \
+  --set providers.azure.vnet.subnetName=$ACI_SUBNET_NAME \
+  --set providers.azure.vent.subnetCidr=$ACI_SUBNET_RANGE \
+  --set providers.azure.vnet.clusterCidr=$CLUSTER_SUBNET_RANGE \
+  --set providers.azure.vnet.kubeDnsIp=$KUBE_DNS_IP \
```

Comment:
remove the last \

Reply:
miss masterUri:
```
MASTER_URI=<AKS public URI, get from kubectl cluster-info>
 --set providers.azure.masterUri=$MASTER_URI
```
Same as below, but **Kubernetes master URI** instead of "AKS public URI"

Reply:
cool will update today! thanks